gh-dependabot

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+import sys
+import click
+import shutil
+import subprocess
+import json
+import csv
+
+@click.group()
+def dependabot():
+    """A GH CLI extension to manage dependabot"""
+
+@dependabot.command()
+@click.argument('repo', nargs=-1)
+@click.option('-o', '--output', help='Path to the output file')
+def export(repo, output):
+    """Pulls all dependabot alerts and exports them to a CSV file
+    
+       REPO is space separated in the OWNER/NAME format
+    """
+    alerts = []
+    for repository in repo:
+        click.echo("Exporting dependabot alerts from %s into %s" % (repository, output))
+
+        alerts += get_dependabot_alerts(repository)
+    
+    generate_csv(alerts, output)
+    
+
+def get_dependabot_alerts(repo_name, cursor='null'):
+    query = 'query ($org: String! $repo: String! $cursor: String){ \
+      repository(owner: $org name: $repo) { \
+        name \
+        vulnerabilityAlerts(first: 100 after: $cursor) { \
+          pageInfo { \
+            hasNextPage \
+            endCursor \
+          } \
+          totalCount \
+          nodes { \
+            id \
+            securityAdvisory { \
+              ...advFields \
+            } \
+            securityVulnerability { \
+              package { \
+                ...pkgFields \
+              } \
+              vulnerableVersionRange \
+            } \
+            createdAt \
+            state \
+            fixedAt \
+            fixReason \
+            dismissedAt \
+            dismissReason \
+            dismisser { \
+                login \
+            } \
+            vulnerableManifestPath \
+            vulnerableRequirements \
+          } \
+        } \
+      } \
+    } \
+    fragment advFields on SecurityAdvisory { \
+      ghsaId \
+      permalink \
+      severity \
+      description \
+      summary \
+    } \
+    fragment pkgFields on SecurityAdvisoryPackage { \
+      name \
+      ecosystem \
+    } \
+    '
+    gh_cli = shutil.which('gh')
+    repo_name_list = repo_name.split('/')
+    command = [ gh_cli, 'api', 'graphql', '-F', "org=%s" % repo_name_list[0], '-F', "repo=%s" % repo_name_list[1],  '-F', "cursor=%s" % cursor, '-f', "query=%s" % query]
+
+    try:
+        process = subprocess.run(command, text=True, capture_output=True, check=True)
+    except subprocess.CalledProcessError as error:
+        click.echo(error.stdout)
```

Comment:
You may want to check the process.returncode/process.stderr here for the subprocess error result, but its probably the same thing.